OpenRA.Game/Traits/World/ActorMap.cs

```diff
@@ -88,24 +86,20 @@ public void Tick(ActorMap am)
 
 		class ProximityTrigger : IDisposable
 		{
-			public readonly int Id;
-			public WPos Position { get; private set; }
-			public WDist Range { get; private set; }
```

Comment:
This is useful info that makes sense to be public, even if nothing is using it right now.